src/agent/onefuzz-supervisor/src/agent.rs

```diff
@@ -44,7 +44,7 @@ impl Agent {
             setup_runner,
             work_queue,
             worker_runner,
-            _heartbeat: heartbeat,
+            heartbeat: heartbeat,
```

Comment:
Since the arg and field names now match, you can replace `heartbeat: heartbeat` with `heartbeat`.